### DISCUSSION TITLE
Get the URL of the page where the form was submitted.

### DISCUSSION THREAD

**Discussion by Megafry** [solspace/craft-freeform#597] [Questions & Answers] [answered]
One of the forms is on multiple pages and we need to know from which url the form was submitted, 
Is this information already stored and available to be outputted in the confirmation email template?

Or do we need to create a hidden field and fill it with `overrideValues`?

Thank you for your support

**Comment by kjmartens** [comment, marked as answer]
This is not something that Freeform includes by default, but you are on the right track with using `overrideValues`. Just pair it with a hidden field and it will work. You can see the guide here:
https://docs.solspace.com/craft/freeform/v4/guides/passing-dynamic-data-to-forms.html

Essentially, let's say you create a hidden field with the handle of `currentUrl`, you can add this parameter to your form in the template:

``` twig
overrideValues: {
  currentUrl: url(craft.app.request.pathInfo),
}
```

Hope this helps 🙂
